geom/slippy/tile.go

```diff
@@ -26,49 +30,123 @@ type Tile struct {
 	SRID uint64
 }
 
-func (t *Tile) ZXY() (uint, uint, uint) {
-	return t.z, t.x, t.y
+func (t *Tile) ZXY() (uint, uint, uint) { return t.z, t.x, t.y }
+
+func Tile2Lon(x, z uint) float64 { return float64(x)/math.Exp2(float64(z))*360.0 - 180.0 }
```

Comment:
Do we know what point of the tile is being returned (i.e. lower left, upper right)?

Reply:
Per definition it will be the upper left.

Reply:
Ok. Consider adding a comment in the future. I'm going to approve this PR